Havit.Blazor.Components.Web.Bootstrap.Documentation/Pages/Components/HxSidebarDoc/HxSidebar_Demo_Logo.razor

```diff
@@ -0,0 +1,34 @@
+﻿<HxSidebar ResponsiveBreakpoint="SidebarResponsiveBreakpoint.None">
```

Comment:
Why do we have the `ResponsiveBreakpoint` here?
...we should not mix several aspects into single demo where it does not add any value.